Mathlib/CategoryTheory/Monoidal/OfChosenFiniteProducts/Basic.lean

```diff
@@ -78,6 +78,28 @@ def BinaryFan.braiding {X Y : C} {s : BinaryFan X Y} (P : IsLimit s) {t : Binary
     (Q : IsLimit t) : s.pt ≅ t.pt :=
   IsLimit.conePointUniqueUpToIso P Q.swapBinaryFan
 
+section
+
+variable {X Y : C} {s : BinaryFan X Y} (P : IsLimit s) {t : BinaryFan Y X} (Q : IsLimit t)
+
+@[reassoc (attr := simp)]
```

Comment:
I do not understand how this can be a simp lemma as `s` does not appear in the LHS?

Reply:
It's in the type of `P`?

Reply:
Yes, as Jakob says. You can look at `braiding_hom_fst` in the other file for an example where `simp` applies this lemma successfully.

Reply:
Ok, I was confused because of the capitalization of the variables...